www/content/workstation/getting_started.md

```diff
@@ -2,14 +2,14 @@
 title = "Getting Started"
 draft = false
 
-aliases = ["getting_started.html", "/workstation_setup.html", "/chefdk_setup.html", "/workstation.html", "/getting_started/"]
```

Comment:
In `install_workstation.md` I linked to `[Getting Started](/workstation/getting_started/)` - will that break now?

Reply:
Nope! That link worked fine in my local build. Although http://localhost:1313/workstation/workstation_setup/ gives me a 404. I apparently do not understand what these aliases mean :)

Reply:
The alias adds a blank page at that url with a redirect to the page that specifies the alias.